<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Get_Button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(129, 255, 120);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Search_LineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>290</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Поиск</string>
    </property>
    <property name="whatsThis">
     <string>Поиск</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(200, 200, 200);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="placeholderText">
     <string>Город</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time_Label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(200, 200, 200);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="text">
     <string>Время</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Geo_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(200, 200, 200);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="text">
     <string>Геопозиция</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Temp_Label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(200, 200, 200);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="text">
     <string>Темп.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Weather_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>190</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background-color: rgb(200, 200, 200);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="text">
     <string>Погода</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Photo_Label">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>70</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Window_Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>410</width>
      <height>135</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#Window_Label{
	
	background-color: rgb(90, 90, 90);
	border-radius: 10px;
	color: rgb(70, 70, 70);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Button">
    <property name="geometry">
     <rect>
      <x>378</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 8px;
	
	background-color: rgb(255, 107, 107);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <zorder>Window_Label</zorder>
   <zorder>Get_Button</zorder>
   <zorder>Search_LineEdit</zorder>
   <zorder>Time_Label</zorder>
   <zorder>Geo_Label</zorder>
   <zorder>Temp_Label</zorder>
   <zorder>Weather_Label</zorder>
   <zorder>Photo_Label</zorder>
   <zorder>Exit_Button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
